evap/evaluation/models.py

```diff
@@ -958,6 +959,23 @@ def annotate_with_participant_and_voter_counts(cls, evaluation_query):
             num_voters=Subquery(voter_count_subquery),
         )
 
+    @classmethod
+    def get_sorted_evaluation_url_tuples_with_urgent_review(cls) -> List[Tuple["Evaluation", str]]:
+        evaluation_url_tuples = [
+            (
+                evaluation,
+                PAGE_URL
+                + reverse(
+                    "staff:evaluation_textanswers",
```

Comment:
Not sure, but maybe this whole method should be a free function in the file where it is used, it feels very specific to the one use case. For example, currently the name suggests that there is one kind of "url" that is associated with an evaluation, the url returned is actually related to textanswer review / staff things though.  The type annotation could then also be changed to remove the quotes :) What do you think?